docs/source/app-dev/authorization.rst

```diff
@@ -7,124 +7,189 @@ Authorization
 #############
```

Comment:
Maybe this is not the right section for this but reading through it it seemed to be missing some instructions for how you actually use this on the gRPC API or the JSON API. Doesn’t have to be directly in here but I think at least some pointers would be very helpful.

Reply:
Good point. I've added this paragraph here: https://github.com/digital-asset/daml/pull/12398/commits/be09ee4d8a7fb57f34ea52b8fb98bda81dff604d

I'd hope that @stefanobaghino-da and his team can add further pointers in later PRs as part of their work on the docs for the JSON-API, auth-middleware, and other tools that are being adapted.